devices/uefi-x86_64/default.nix

```diff
@@ -24,9 +24,11 @@
 
   mobile.boot.stage-1 = {
     kernel = {
-      package = pkgs.linuxPackages_5_4.kernel
-        .overrideAttrs({passthru ? {}, ...}: {
-          passthru = passthru // {
+      package = let inherit (pkgs.linuxPackages_5_4) kernel; in
+        kernel.overrideAttrs({passthru ? {}, ...}: {
+          # `kernel` needs to be merged here, otherwise we seemingly lose the
+          # `features` attribute which will be needed later on.
+          passthru = kernel // passthru // {
             file = "bzImage";
           };
```

Comment:
I don't get why I needed to do this to get `features` to be available again on the kernel attribute.

To test:

```
 $ nix-instantiate ./default.nix --argstr device uefi-x86_64 -A build.toplevel
```